debian/rules

```diff
@@ -12,9 +13,3 @@ override_dh_python3:
 
 override_dh_missing:
 	rm -r `find debian/tmp -name __pycache__`
```

Comment:
this is unusual, that dh_missing target is overriden, and dh_missing is not at all called.

Did you mean to have that as
```
 execute_before_dh_missing:
    rm -r find debian/tmp -name __pycache__
```

Reply:
@xnox Yes, should have been `execute_before_dh_missing` or `execute_after_something_else`. Thanks for spotting this.